oas-validator/README-ZH.md

```diff
@@ -8,154 +8,139 @@ OpenAPI V3 Spec校验工具。
 * oas-validator-core-spring，骨架的Spring Boot Starter
 * oas-validator-test，核心API的测试帮助类
 * oas-validator-compliance，合规性校验实现
-* oas-validator-compliance-spring，合规性校验的Spring Boot Starter
```

Comment:
why did you remove this line.

Reply:
oas-validator-compliance-spring directly @Configure *Validator beans, which are not configurable. While we use *ValidatorFactory now, so we can abandon it.

Related commit:
* https://github.com/apache/servicecomb-toolkit/pull/52/commits/ece6c372630deb4b24ddbfb64fd389eba4cb3572
* https://github.com/apache/servicecomb-toolkit/pull/52/commits/9c9c22f03e20362b976e067c3d67e589988db727